integrated_channels/sap_success_factors/utils.py

```diff
@@ -253,7 +253,8 @@ def resolve_removed_courses(self, previous_audit_summary):
                     'sap_success_factors',
                     'SAPSuccessFactorsGlobalConfiguration'
                 ).current().provider_id,
-                'launchURL': get_course_track_selection_url(x['enterprise_customer'], x['key']),
+                'launchURL': (get_enterprise_course_enrollment_url(x['enterprise_customer'], x['key']) or
+                              get_course_track_selection_url(x['enterprise_customer'], x['key'])),
```

Comment:
I'm wondering if we should waffle switch this. I don't think you would ever get to the second clause of the `or` statement here would you?